docs/sdk/access-control/other-chains/stellar-conditions.md

```diff
@@ -0,0 +1,317 @@
+# Custom Access Control for Stellar
+
+The guide will walk you through an implementation of custom authentication and authorization using Lit Actions for the Stellar blockchain. Our goal is to authenticate who we are (by signing a Stellar transaction), and make a call to a Stellar smart contract to check if our authenticated address is on an allow list.
+
+The full implementation of this code example is available [here](https://github.com/LIT-Protocol/developer-guides-code/blob/master/lit-access-control-conditions-stellar/nodejs). Instruction on running the example are provided in the code example's `README.md`.
+
+## Signing a Stellar Transaction for Authentication
+
+The following code for this section can be found [here](https://github.com/LIT-Protocol/developer-guides-code/blob/master/lit-access-control-conditions-stellar/nodejs/src/index.ts)
+
+The first step in our code example is authenticating the address that we're going to check the allow list for. To do this, we're going to initialize a Stellar wallet:
+
+```js
+const stellarKeyPair = StellarBase.Keypair.fromSecret(STELLAR_SECRET);
+const stellarAccount = new StellarBase.Account(
+    stellarKeyPair.publicKey(),
+    STELLAR_ACCOUNT_SEQUENCE_NUMBER
+);
+```
+
+and then sign an empty Stellar transaction:
+
+```js
+const stellarAuthTx = new StellarBase.TransactionBuilder(stellarAccount, {
+    fee: StellarBase.BASE_FEE,
+    networkPassphrase: StellarBase.Networks.TESTNET,
+})
+    .setTimeout(60 * 60 * 24) // 24 hours
+    .build();
+stellarAuthTx.sign(stellarKeyPair);
+```
+
+This signed transaction is what we're going to submit to our Lit Action in order to authenticate an address.
+
+## Calling our Lit Action
+
+### Connecting to the Lit Network
+
+The first step to calling our Lit Action is to connect to the Lit Network:
+
+```js
+litNodeClient = new LitNodeClientNodeJs({
+    litNetwork: LitNetwork.Cayenne,
+});
+await litNodeClient.connect();
+```
+Once we have a connected client, the next step is authenticating and authorizing ourselves to use the Lit Network. To do this, we're going to instantiate an instance of ethers.js `Wallet`:
+
+```js
+const ethersWallet = new Ethers.Wallet(
+    ETHEREUM_PRIVATE_KEY,
+    new Ethers.providers.JsonRpcProvider(
+        "https://chain-rpc.litprotocol.com/http"
+    )
+);
+```
+
+And then create a [Session Signature](../../../sdk/authentication/session-sigs/intro.md):
```

Comment:
Does this only work on Cayenne? Will this guide be followable if they're using Manzano/Habanero? If not, we should provide a link to some static doc (Maybe the "Migrating from earlier versions" page?) where they can learn all the changes needed to get it working on those networks and, eventually Datil so we don't have to update this again.